File: CS/RegisterFromLogonFormSolution.Module.Web/WebModule.cs
using System;
using System.ComponentModel;

using DevExpress.ExpressApp;
using WinWebSolution.Module.Web.Controllers;

namespace RegisterFromLogonFormSolution.Module.Web {
    [ToolboxItemFilter("Xaf.Platform.Web")]
    public sealed partial class RegisterFromLogonFormSolutionAspNetModule : ModuleBase {
        public RegisterFromLogonFormSolutionAspNetModule() {
            InitializeComponent();
        }
        public override void Setup(XafApplication application) {
            base.Setup(application);
            application.CreateCustomLogonWindowControllers += application_CreateCustomLogonWindowControllers;
        }
        private void application_CreateCustomLogonWindowControllers(object sender, CreateCustomLogonWindowControllersEventArgs e) {
            XafApplication app = (XafApplication)sender;
            e.Controllers.Add(app.CreateController<WebLogonActionParametersController>());
        }
    }
}

File: VB/RegisterFromLogonFormSolution.Module.Web/WebModule.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel

Imports DevExpress.ExpressApp
Imports WinWebSolution.Module.Web.Controllers

Namespace RegisterFromLogonFormSolution.Module.Web
	<ToolboxItemFilter("Xaf.Platform.Web")> _
	Public NotInheritable Partial Class RegisterFromLogonFormSolutionAspNetModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub
		Public Overrides Overloads Sub Setup(ByVal application As XafApplication)
			MyBase.Setup(application)
			AddHandler application.CreateCustomLogonWindowControllers, AddressOf application_CreateCustomLogonWindowControllers
		End Sub
		Private Sub application_CreateCustomLogonWindowControllers(ByVal sender As Object, ByVal e As CreateCustomLogonWindowControllersEventArgs)
			Dim app As XafApplication = CType(sender, XafApplication)
			e.Controllers.Add(app.CreateController(Of WebLogonActionParametersController)())
		End Sub
	End Class
End Namespace
